environments/ithc/ithc-platform-hmcts-net.yml

```diff
@@ -4,5 +4,69 @@ vnet_links:
     vnet_id: "/subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c/resourceGroups/core-infra-ithc/providers/Microsoft.Network/virtualNetworks/core-infra-vnet-ithc"
   - link_name: "core-infra-vnet-idam-ithc-internal"
     vnet_id: "/subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c/resourceGroups/core-infra-idam-ithc/providers/Microsoft.Network/virtualNetworks/core-infra-vnet-idam-ithc"
-A: []
-cname: []
+A:
+  - name: ccd-admin-web
+    record:
+    - 10.10.36.121
+    ttl: 300
+cname:
+  - name: neuvector
+    record: 10.10.36.6.ithc.platform.hmcts.net.
```

Comment:
This doesn’t look right?

Reply:
Well spotted. It comes from this: - {name: 'neuvector.ithc', ttl: 300, type: 'CNAME', dest: '10.10.36.6'}
So it's not a but in my conversion script :-)

Reply:
so it's a rubbish record 😄